homeassistant/components/tts/voicerss.py

```diff
@@ -0,0 +1,51 @@
+"""
+Support for VoiceRSS TTS in Home Assistant.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/tts.voicerss/
+"""
+
+import logging
+import shutil
+import urllib.parse
+import requests
+
+from homeassistant.helpers import validate_config
+from homeassistant.components.tts import (DOMAIN, BaseTTSService)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_engine(hass, config):
+    """Get the VoiceRSS TTS service."""
+    if not validate_config({DOMAIN: config},
+                           {DOMAIN: ['key']},
+                           _LOGGER):
+        return None
+
+    return VoiceRSSService(config['key'])
```

Comment:
Only by burning an API request of which there's a monthly limit. I can implement it if you want though.